tutorial6_hierarchical/demo/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+  <head>
+    <meta charset="utf-8">
+
+    <!-- this is the title that goes in the browser tab name  -->
+    <title>Interactive Data Visualization Tutorial</title>
+
+    <!-- custom font: https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap -->
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header>
+      <h1>Tutorial 6</h1>
+      <h2>Hierarchical</h2>
+    </header>
+
+    <!-- all we need is an element to select -->
+    <div id="d3-container"> </div>
+
+    <!-- this is the line that loads in d3 -->
+    <script src="../../lib/d3.js"></script>
+    <script src="../../lib/d3-array.js"></script>
```

Comment:
we don't necessarily need this if we're not using `rollups` but might be nice to have just in case